docs/robot.md

```diff
@@ -0,0 +1,138 @@
+# Clusters with Robot Servers
+
+## Quickstart
+
+Prerequisites:
+
+- Running Cluster
+- Kubectl
+- Helm
+
+0. Make sure that you start all Kubelets in the cluster with `--cloud-provider=external`.
+
+1. Export credentials and create secret:
+
+   ```bash
+    export HCLOUD_TOKEN=your-hcloud-token
+    export ROBOT_USER=your-robot-user-name
+    export ROBOT_PASSWORD=your-robot-password
+   kubectl -n kube-system create secret generic hcloud --from-literal=token=$HCLOUD_TOKEN --from-literal=robot-user=$ROBOT_USER --from-literal=robot-password=$ROBOT_PASSWORD
+   ```
+
+2. Install the Helm Chart:
+
+   ```bash
+   helm repo add hcloud https://charts.hetzner.cloud
+   helm repo update hcloud
+   helm install hcloud/hcloud-cloud-controller-manager --set robot.enabled=true
+   ```
+
+You should now see that the Robot Server was initialized and the labels added.
```

Comment:
Which labels are added where? As newcomer this is not obvious.

Reply:
The labels are described in the section _Node Controller_, I will add a link.